Build.cmd

```diff
@@ -137,6 +141,11 @@ editbin /NOLOGO /LARGEADDRESSAWARE "Program\RunActivityLAA.exe" || GOTO :error
 copy "Program\RunActivity.exe.config" "Program\RunActivityLAA.exe.config" || GOTO :error
 ECHO Created large address aware version of RunActivity.exe.
 
+REM Copy the Web content, empty the destination folder first
+IF EXIST "Program\Content\Web" RMDIR "Program\Content\Web" /S /Q
+IF NOT EXIST "Program\Content\Web" MKDIR "Program\Content\Web"
```

Comment:
The utility ":recreate" should do these two steps for you, e.g.

```
CALL :recreate "Program\Content\Web" || GOTO :error
```

However, because the entire `Program` directory is recreated earlier on line 102/106, this whole bit is not needed.